index.html

```diff
@@ -12,67 +12,206 @@
     font-size: 12px;
     color: #111111;
 }
-#solIcon {
-    right: 10px;
-    top: 10px;
+#editor {
     position: absolute;
-    height: 100px;
-    width: 100px;
-    overflow: hidden;
-}
-#solIcon img {
-    top: -20px;
-    right: -10px;
-    position: absolute;
-    height: 100px;
-}
-#optimizeBox {
-    position: absolute;
-    top: 30px;
-    left: 720px;
-    font-size: 14px;
-}
-#input {
-    position: absolute;
-    top: 120px;
+    top: 0;
     left: 0px;
-    width: 700px;
+    width: auto;
     bottom: 0px;
+    right: 37em;
+    
+}
+#input {
     font-size: 15px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    min-width: 20vw;
 }
-#output {
+
+#righthand-panel {
     position: absolute;
-    top: 120px;
-    left: 720px;
-    right: 10px;
+    top: 0;
+    width: 37em;
```

Comment:
Is that large enough? On my screen, I have about 3/4 editor and 1/4 righthand panel, and I would actually prefer the editor size to be at least 90 chars (about the position of the vertical line in the editor) and the righthand panel at least 1/3 of the screen or something like that.
But that is only a minor thing.